### PR TITLE
Added Golang client

### PR DESCRIPTION
**Pull request by AlessandroSechi**

### CODE REVIEW

File: api/intro.rst [hunk 1]
```diff
@@ -22,6 +22,8 @@ endpoints listed here.
    * `.NET Client <https://github.com/Asesjix/Zammad-Client>`_ *(Third-Party)*
    * `Android API-Client <https://github.com/KirkBushman/zammad-android>`_ 
      *(Third-Party)*
+   * `Golang Client <https://github.com/AlessandroSechi/zammad-go>`_
```
Comment (MrGeneration) [inline api/intro.rst:25]: Wouldn't be "Go client" a better wording?
At leas thtat's what I guess from the Readme of the repo in question.

If not just let me know. :)

Comment (MrGeneration) [inline api/intro.rst:25]: Wouldn't be "Go client" a better wording?
At leas thtat's what I guess from the Readme of the repo in question.

If not just let me know. :)

**Review by MrGeneration — CHANGES_REQUESTED** [review]

File: api/intro.rst (hunk 1 above)
Comment (AlessandroSechi) [reply api/intro.rst]: Yes it is. I fixed it

**Review by MrGeneration — CHANGES_REQUESTED** [review]
Thank you very much for your contribution!

**Review by MrGeneration — APPROVED** [review]

**Pull request merged**